tests/apps/courses/test_models_course.py

```diff
@@ -24,6 +28,64 @@ def test_models_course_str(self):
         with self.assertNumQueries(1):
             self.assertEqual(str(course), "Course: Nano particles")
 
+    def test_models_course_get_organizations_empty(self):
+        """
+        For a course not linked to any organzation the method `get_organizations` should
+        return an empty query.
+        """
+        course = CourseFactory(should_publish=True)
+        self.assertFalse(course.get_organizations().exists())
+        self.assertFalse(course.public_extension.get_organizations().exists())
+
+    def test_models_course_get_organizations(self):
+        """
+        The `get_organizations` method should return all organizations linked to a course and
+        should respect publication status.
+        """
+        *draft_organizations, _other_draft = OrganizationFactory.create_batch(3)
```

Comment:
At this moment yes. But in the code that follows I use `draft_organizations` to link them to the course whereas `_other_draft` ends up not being linked to any course. 

This is to test that organizations are returned because they are linked to the course and not all organizations are returned.

Reply:
Oh I see, hence the _ prefix as it’s just for unpacking. Maybe this needs a comment to make sure no one removes it by mistake